src/ppom-controller.ts

```diff
@@ -17,6 +17,7 @@ export const REFRESH_TIME_DURATION = 1000 * 60 * 60 * 24;
 
 const PROVIDER_REQUEST_LIMIT = 500;
 const MILLISECONDS_IN_FIVE_MINUTES = 300000;
+const MILLISECONDS_IN_ONE_WEEK = 604800000;
```

Comment:
Very minor, but for duration constants like this, would it help maintenance and readability to include the multiplication such as:

```
const MILLISECONDS_IN_ONE_WEEK = 1000 * 60 * 60 * 24 * 7;
```

Reply:
Also, would it add clarity to name these based on usage rather than their literal value, such as `DATA_REFRESH_INTERVAL`?

Reply:
Done